content/millennium/r4/diagnostic/observation.md

```diff
@@ -246,9 +246,9 @@ Update an existing Observation.
 
 _Implementation Notes_
 
-* Currently only `laboratory` is supported.
+* Currently only `laboratory` and `vital-signs` is supported.
 * Both read and write scopes are required.
-* Updates on patient and category are not supported. Updating a value to a different value type is not supported.
+* Updates on patient and category are not supported.
 * See [FHIR<sup>®</sup> Update] for additional details about update operations.
```

Comment:
Can you include the note we included in creates about active relationship here too, please?

Reply:
<img width="702" alt="Screen Shot 2021-06-18 at 9 28 26 AM" src="https://user-images.githubusercontent.com/18075594/122576717-98202200-d017-11eb-9b3c-43106a34f575.png">